src/lang/Messages.properties

```diff
@@ -71,6 +71,7 @@ api.dialogue.browseApiNotEnabled.title = API Not Enabled
 api.dialogue.browseApiNotEnabled.message = The API must be enabled before you can browse it.\nDo you want to enable the API?
 api.dialogue.browseApiNotEnabled.button.confirm.label = Enable and Browse the API
 api.dialogue.browseApiNotEnabled.button.cancel.label = Cancel
+api.deprecated.option.endpoint = Option no longer in effective use.
```

Comment:
I think we need a different word here, "effective" implies that it can be used but that the current use is partially wrong or something like that. i.e.: If it is no longer being effectively used then it was at some point, or could be at some point in the future.